samples/compose-resources-gallery/androidApp/build.gradle.kts

```diff
@@ -24,8 +24,13 @@ android {
         versionCode = 1
         versionName = "1.0"
     }
-    compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_11
-        targetCompatibility = JavaVersion.VERSION_11
+
+    namespace = "com.myapplication"
+}
+
+afterEvaluate {
+    tasks.withType<JavaCompile>().configureEach {
+        sourceCompatibility = JavaVersion.VERSION_17.toString()
+        targetCompatibility = JavaVersion.VERSION_17.toString()
```

Comment:
why not use old variant of setup? without `afterEvaluate`. and why we should use 17?